core/string/ustring.cpp

```diff
@@ -1016,22 +1016,20 @@ String String::get_slicec(char32_t p_splitter, int p_slice) const {
 		return String();
 	}
 
-	const char32_t *c = this->ptr();
 	int i = 0;
 	int prev = 0;
 	int count = 0;
 	while (true) {
-		if (c[i] == 0 || c[i] == p_splitter) {
+		if (i == length() || this->ptr()[i] == p_splitter) {
```

Comment:
In debug builds, calling `this->ptr()[]` on each step is 3...5 times slower than using `const char32_t *c = this->ptr();` once!